patches/server/0040-Add-packet-limiter-config.patch

```diff
@@ -134,8 +143,30 @@ index cb33129ac32e767ab5e5ffaec77ddd90cc9536ae..0e331eb38b91d6323bd64bddc2a3274f
      public static final class WorldConfig {
  
          public final String worldName;
+diff --git a/src/main/java/io/papermc/paper/util/StacktraceDeobfuscator.java b/src/main/java/io/papermc/paper/util/StacktraceDeobfuscator.java
+index 281fd2dbc7f1281ba882020be2a3481fe8909685..8f663eeb773adfae71288880bdb2b7da32be0bf3 100644
+--- a/src/main/java/io/papermc/paper/util/StacktraceDeobfuscator.java
++++ b/src/main/java/io/papermc/paper/util/StacktraceDeobfuscator.java
+@@ -32,7 +32,7 @@ public enum StacktraceDeobfuscator {
+     private static final String MOJANG_PLUS_YARN_NAMESPACE = "mojang+yarn";
+     private static final String SPIGOT_NAMESPACE = "spigot";
+ 
+-    private final @Nullable Map<String, ClassMapping> mappings;
++    private final @Nullable Map<String, ClassMapping> mappings; public @Nullable Map<String, ClassMapping> mappings() { return this.mappings; }; // Tuinity - public getter
+     private final Map<Class<?>, Map<Pair<String, String>, IntList>> lineMapCache = Collections.synchronizedMap(new LinkedHashMap<>(64, 0.75f, true) {
+         @Override
+         protected boolean removeEldestEntry(final Map.Entry<Class<?>, Map<Pair<String, String>, IntList>> eldest) {
+@@ -207,7 +207,7 @@ public enum StacktraceDeobfuscator {
+         return rootClassName + ".java";
+     }
+ 
+-    private record ClassMapping(
++    public record ClassMapping( // Tuinity - private -> public
```

Comment:
can't we get use of paperweight's access transformers or something?
I see by the paperweight examples an access transformer file isn't included there, but why?